examples/jupyter_lite_config.json

```diff
@@ -59,24 +58,26 @@
       "https://files.pythonhosted.org/packages/py3/b/branca/branca-0.4.2-py3-none-any.whl",
       "https://files.pythonhosted.org/packages/py3/c/charset-normalizer/charset_normalizer-2.0.12-py3-none-any.whl",
       "https://files.pythonhosted.org/packages/py3/e/entrypoints/entrypoints-0.4-py3-none-any.whl",
+      "https://files.pythonhosted.org/packages/e6/0b/24795939622d60f4b453aa7040f23c6a6f8b44c7c026c3b42d9842e6cc31/fastjsonschema-2.15.3-py3-none-any.whl",
```

Comment:
it would be annoying if we had to do special handling for this here on out, but I could not construct a "predictable" url for it, yet...

Reply:
Why not https://files.pythonhosted.org/packages/py3/f/fastjsonschema-2.15.3-py3-none-any.whl ?

Reply:
Ahh, that's 0kb. Weird.

Reply:
AFAICT, you are supposed to use the static URLs and REST queries. The URL can be obtained from:

```console
$ curl https://pypi.org/pypi/fastjsonschema/json | jq '.info.version'
"2.15.3"
$ curl https://pypi.org/pypi/fastjsonschema/json | jq '.releases | .["2.15.3"] | .[].url'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 26619  100 26619    0     0  89540      0 --:--:-- --:--:-- --:--:-- 92107
"https://files.pythonhosted.org/packages/e6/0b/24795939622d60f4b453aa7040f23c6a6f8b44c7c026c3b42d9842e6cc31/fastjsonschema-2.15.3-py3-none-any.whl"
"https://files.pythonhosted.org/packages/50/1e/44bf3d37e60f36af7c3f1462425082c2d925080bb9926a32c3b8439e67de/fastjsonschema-2.15.3.tar.gz"
```

Reply:
yep, am aware of that api, as we end up having to partially emulate it. i prefer the predictable ones for documentation/build purposes.

Reply:
AFAICT, the "python_version" part, which is py3 in the other ones, is supposed to be `"3.7"`, but that's not working in any variation I tried.

Reply:
yerp, semi-defeats the predictable part, so have given up and used the api  in 5c5e0c9